try {
    let a ;
    console.log(a.length);
    console.log("Hi there from inside");
}
catch(e){
    console.log("Inside cactch statement");
}

console.log("HI there")